src/main/kotlin/no/nav/syfo/sykmelding/kafka/service/SykmeldingStatusConsumerService.kt

```diff
@@ -59,24 +63,39 @@ class SykmeldingStatusConsumerService(
     }
 
     private fun handleStatusEvent(): (SykmeldingStatusKafkaMessageDTO) -> Unit = { sykmeldingStatusKafkaMessage ->
-        log.info("Got status update from kafka topic, sykmeldingId: {}, status: {}", sykmeldingStatusKafkaMessage.kafkaMetadata.sykmeldingId, sykmeldingStatusKafkaMessage.event.statusEvent.name)
+        log.info("Got status update from kafka topic, sykmeldingId: {}, status: {}", sykmeldingStatusKafkaMessage.kafkaMetadata.sykmeldingId, sykmeldingStatusKafkaMessage.event.statusEvent)
         try {
             when (sykmeldingStatusKafkaMessage.event.statusEvent) {
-                StatusEventDTO.SENDT -> {
+                STATUS_SENDT -> {
                     handleSendtSykmelding(sykmeldingStatusKafkaMessage)
                 }
-                StatusEventDTO.BEKREFTET -> {
+                STATUS_BEKREFTET -> {
                     registrerBekreftet(sykmeldingStatusKafkaMessage)
                     publishToBekreftSykmeldingTopic(sykmeldingStatusKafkaMessage)
                 }
+                STATUS_SLETTET -> {
```

Comment:
Er det ok for konsumentene at vi legger til en status? :)

Reply:
Ja det har jeg sjekket før sommern, men kan dobbeltsjekke egentlig

Reply:
Ja, stemmer, husker at du sjekket det nå, når jeg tenker meg om :D